src/components/common/applicant-info-1.vue

```diff
@@ -589,13 +589,15 @@ export default class ApplicantInfo1 extends NameRequestMixin {
 
   @Watch('isValid')
   onValidChanged (val: boolean) {
-    if (val && this.$el?.querySelector) {
+    if (val) {
       this.$nextTick(() => {
-        // add classname to button text (for more detail in Sentry breadcrumbs)
-        const applicantBackBtn = this.$el.querySelector("#submit-back-btn > span")
-        if (applicantBackBtn) applicantBackBtn.classList.add("applicant-back-btn")
-        const applicantContinueBtn = this.$el.querySelector("#submit-continue-btn > span")
-        if (applicantContinueBtn) applicantContinueBtn.classList.add("applicant-continue-btn")
+        if (this.$el?.querySelector) {
```

Comment:
My hope here is that, in the next tick, $el and querySelector exist (because for some unknown reason, they don't initially, which is why this check is here).